### ISSUE TITLE
Odd output not from realize

### ISSUE THREAD

**Issue by SpencerCDixon** [tockins/realize#105]
I see very odd ERROR output when I start realize that doesn't have to do with my project.  Has anyone else seen this:

```sh
{"time":"2017-10-24T16:07:50.858682348-04:00","level":"ERROR","prefix":"echo","file":"echo.go","line":"284","message":"code=404, message=Not Found"}
{"time":"2017-10-24T16:07:50.868707701-04:00","level":"ERROR","prefix":"echo","file":"echo.go","line":"284","message":"code=404, message=Not Found"}
{"time":"2017-10-24T16:07:50.871328904-04:00","level":"ERROR","prefix":"echo","file":"echo.go","line":"284","message":"code=404, message=Not Found"}
{"time":"2017-10-24T16:07:51.268499704-04:00","level":"ERROR","prefix":"echo","file":"echo.go","line":"284","message":"code=404, message=Not Found"}
{"time":"2017-10-24T16:07:51.270975077-04:00","level":"ERROR","prefix":"echo","file":"echo.go","line":"284","message":"code=404, message=Not Found"}
{"time":"2017-10-24T16:07:51.282964849-04:00","level":"ERROR","prefix":"echo","file":"echo.go","line":"284","message":"code=404, message=Not Found"}
```

**Comment by SpencerCDixon** [comment]
Looks like it's definitely coming from the dev server that comes with Realize.  Not sure how to get it to stop though. Perhaps an error in the vendored packages Realize uses?

**Comment by mingrammer** [comment]
Those messages from echo which is a Go web framework realize uses

**Comment by Asoseil** [comment]
We will change the web framework as soon as possible. Anyway a temporary fix will come with v1.5.1

**Comment by Asoseil** [comment]
#106
